schema/elasticsearch/visibility/versioned/v1/reindex.sh

```diff
@@ -16,25 +23,43 @@ AUTO_CONFIRM="${AUTO_CONFIRM:-}"
 ES_ENDPOINT="${ES_SCHEME}://${ES_SERVER}:${ES_PORT}"
 DIR_NAME="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
 
+if ! curl --silent --user "${ES_USER}":"${ES_PWD}" "${ES_ENDPOINT}/_cluster/health" | jq --exit-status '.status=="green" | .'; then
```

Comment:
minor but you probably want --fail here

Reply:
`--silent` should include it, no?

Reply:
I don't think so. `curl --silent` will still given an exit code of zero for a 404. `curl --fail` will exit with 22.